test/inductor/test_distributed_patterns.py

```diff
@@ -118,6 +120,32 @@ def steps(m, inp):
     return out
 
 
+fw_graph = [None]
+bw_graph = [None]
+
+
+def aot_graph_capture_backend(gm, args):
+    from functorch.compile import min_cut_rematerialization_partition
+    from torch._functorch.aot_autograd import aot_module_simplified
+
+    def fw_compiler(gm, _):
+        fw_graph[0] = gm
+        return gm
+
+    def bw_compiler(gm, _):
+        bw_graph[0] = gm
+        return gm
+
+    return aot_module_simplified(
+        gm,
+        args,
+        fw_compiler,
+        bw_compiler,
+        partition_fn=min_cut_rematerialization_partition,
+        keep_inference_input_mutations=True,
```

Comment:
if these are non-default options, can you put comments as to why we want them in particular in the distributed tests?

Reply:
yeah these are all the same defaults that vanilla torch.compile uses (annoying boilerplate though)